javascript/javascript1/week1/pizza.js

```diff
@@ -0,0 +1,24 @@
+// main exercise: part one
+console.log('I love pizza');
+let myFavoritePizza = 'pepperoni'
+let pizzaPrice = 50
+console.log(` New pizza order: ${myFavoritePizza}. The price of the pizza is: ${pizzaPrice}`);
+// part one end
+
+// part two
```

Comment:
Let's talk about this one 🤙

Reply:
Yes of course. but when ?

Reply:
I have written a DM on Slack, let's continue there